src/osemgrep/core/Metrics_.ml

```diff
@@ -136,6 +136,8 @@ let default_payload =
     environment =
       {
         version = Version.version;
+        os = Sys.os_type;
```

Comment:
oh no :( 

looks like ocaml doesn't seem to distinguish mac vs linux out-of-the-box.

see https://github.com/ocaml/opam/issues/3058 

(also hannes is on the thread 😆)

Reply:
ugh annoying, we'll just have to fix it in metabase or smthn